tools/tensorflow/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.11
```

Comment:
Interesting, so did you switch to this from `-slim` because it fixed the issues you were having?

Reply:
I tried slim early on but it seemed to cause a problem with tensorflow from the beginning, what fixed it was all the previous imports meant I no longer needed to import -aarch64 and could go with the standard that ended up being compatible with both.

Reply:
OK! So just to clarify, are you saying that you need the non-slim version for it to work correctly? Or are you saying that it'd work now if you switch it back to slim?

Reply:
You need non-slim on mac but I think the slim version does work on x86. I could change the Dockerfile to download slim for x86 if you think it would be worthwhile.

Reply:
Nope that's fine! If ARM needs non-slim then let's use that. Thanks!